src/shared/components/event-meta/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react';
+import styles from './style.css';
+
+import TagsList from '../tags-list';
+import EventLinksList from '../event-links-list';
+
+const EventMeta = ({
+  event,
+}) => (
+  <div>
+  {
+    event.externalLinks || event.internalLinks ?
+      <div className={styles.eventLinks}>
+      {
+        event.externalLinks ?
+          <EventLinksList
+            linkList={event.externalLinks}
+            listType="external" />
+          : null
+      }
+      {
+        event.internalLinks ?
+          <EventLinksList
+            linkList={event.internalLinks}
+            listType="internal" />
+          : null
+      }
+      </div>
+    : null
+  }
+  {
+    event.tags
+    ? <TagsList
+        tags={event.tags}
+        tagsLinkPath="about-us/events" />
+    : null
```

Comment:
Note that we need to update to React 15 in order to get that `null` returns working without warnings.